search/searcher/search_phrase.go

```diff
@@ -285,10 +297,14 @@ func (p phrasePath) MergeInto(in search.TermLocationMap) {
 // returns slice of paths, or nil if invocation did not find any successul paths
 func findPhrasePaths(prevPos uint64, ap search.ArrayPositions, phraseTerms [][]string,
 	tlm search.TermLocationMap, p phrasePath, remainingSlop int) []phrasePath {
+	return findPhrasePathsRecur(prevPos, ap, phraseTerms, tlm, p, remainingSlop, nil)
```

Comment:
since we're introducing the wrapper function, can we correct the comments?

Reply:
will do